docs/getting-started/README.md

```diff
@@ -15,6 +15,17 @@ Welcome! Let's get started building with blocks. Blocks are at the core of exten
 
 [Create a Block Tutorial](/docs/getting-started/create-block/README.md) - Learn how to create your first block for the WordPress block editor.
 
+## Learn WordPress Courses
```

Comment:
I would try to guide beginners a bit more by introducing developers to these learning resources but reducing the specific recommended learning resources and instead providing a strong recommendation as a next step. 

_PS: Some of the courses you mention like [Using the WordPress Data Layer](https://learn.wordpress.org/course/using-the-wordpress-data-layer/) & [Registering Block Patterns](https://learn.wordpress.org/workshop/registering-block-patterns/) could be highlighted in the docs sections dedicated to these topics _

Here's my suggestion for this part...

---------------------------------------------------

## Learn WordPress

At [Learn WordPress](https://learn.wordpress.org/) you can find [tutorials](https://learn.wordpress.org/tutorials/?topic=11), [courses](https://learn.wordpress.org/courses/#developing-with-wordpress), and [workshops](https://wordpress.tv/tag/lets-code-series/) to learn more about block development. 

<!-- I would somehow highlight the following paragraph -->
> **Note**
> A great course to get started in Block Development is **[Intro to Block Development: Build Your First Custom Block](https://learn.wordpress.org/course/introduction-to-block-development-build-your-first-custom-block/)**
</br>